### DISCUSSION TITLE
Shared Elements not working in release?

### DISCUSSION THREAD

**Discussion by yepMad** [software-mansion/react-native-reanimated#4192] [Q&A]
When running in development mode I can see shared-elements in action. But when I build it, I don't see it. This is normal?

**Comment by MissieKahsay** [comment]
During development, you may see shared elements in action because of the way the development server is set up. However, when you build your application for production, it's normal to see differences, including the absence of shared elements. This is because the code is compiled and optimized for performance, and doesn't include the development server or other tools used during development.

To ensure that shared elements are included in your production build, you may need to check your build configuration to make sure they're properly included. Additionally, you should ensure that your production environment is set up correctly, using the correct versions of dependencies and configuration settings. By doing so, you can avoid issues like this and ensure that your application runs smoothly in production.
